uisdk/src/main/res/drawable/kh_prebook_destination_icon.xml

```diff
@@ -0,0 +1,11 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="10dp"
+    android:height="11dp"
+    android:viewportWidth="10"
+    android:viewportHeight="11">
+  <path
+      android:pathData="M5,5.5m-3.5,0a3.5,3.5 0,1 1,7 0a3.5,3.5 0,1 1,-7 0"
+      android:strokeWidth="3"
+      android:fillColor="#ffffff"
+      android:strokeColor="#18183F"/>
```

Comment:
@color/kh_uisdk_primary

Reply:
To add to this, shouldn't the fill color be background2? So we don't have a white center in dark mode?

Reply:
Done